pytest-wdl-tests/tests/expr/commands/test_expressions_commands.wdl

```diff
@@ -0,0 +1,133 @@
+version 1.0
+
+struct Person {
+    String name
+    Int age
+}
+
+task exprs {
+
+    Pair[String, String] pair = ("left", "right")
+    Int i = 42
+    Int j = 43
+    Boolean true_ = true
+    Boolean false_ = false
+    Person p = { "name": "Jamie", "age": 4 }
+    Array[String] booleans = [ true, false ]
+    Array[String] booleans_ = [ true_, false_]
+
+    command {
+        echo ~{"string"}
+        echo ~{("string")}
+        echo ~{("left", "right").left}
+        echo ~{pair.left}
+        echo ~{["string"][0]}
+        echo ~{true}
+        echo ~{false}
+        echo ~{!true}
+        echo ~{true_}
+        echo ~{false_}
+        echo ~{!true_}
+        echo ~{42}
+        echo ~{-42}
+        echo ~{+42}
+        echo ~{i}
+        echo ~{+i} # fails in Cromwell 48
```

Comment:
We should probably keep these kinds of comments in a Cromwell issue tracker rather than the openWDL spec repo